### PR TITLE
Feature/BE-12-view-application-summary

### PR DESCRIPTION
**Pull request by logikcode**
Issue:
As an applicant, I should be able to see summary or view detail of job application 

What the reviewer should know:

Created JobSummaryResponse response dto for data transporting
Created  getProfessionalByMemberEmailAddress method for retrieving applicant using email
Implemented endpoint to retrieve applicant jobs summaries
Created retrieveProfessionalSubmissions method for retrieving applicant submissions

### CODE REVIEW

File: Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ProfessionalController.java [hunk 1]
```diff
@@ -49,5 +48,11 @@ public Page<Professional> retrieveApplicants(@PathVariable int pageNo, @PathVari
         return  service.retrieveAllProfessionals(pageNo, pageSize);
 
     }
+
+    @GetMapping("/view-application-summary/{applicantEmail}")
+    public List<JobSummaryResponse> retrieveApplicantJobSummary(@PathVariable String applicantEmail){
+    return  service.retrieveProfessionalSubmissions(applicantEmail);
+    }
+
```
Comment (Jayhmz) [inline Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ProfessionalController.java:56]: Why are you using email of the user and not the user id as your path variable? also you do not need to pass the id as a path variable instead, fetch it from the security context.

File: Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/repository/jpa/ProfessionalJpaRepository.java [hunk 2]
```diff
@@ -10,4 +10,5 @@ public interface ProfessionalJpaRepository extends JpaRepository<Professional, L
 
     @Query("select p from Professional p where p.member.id = :id")
     Professional getProfessional(@Param("id") Long memberId);
+    Professional getProfessionalByMemberEmailAddress(String emailAddress);
 }
```
Comment (Jayhmz) [inline Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/repository/jpa/ProfessionalJpaRepository.java:14]: reflect the above comment on this also. we do not need to show the personal details of user on the request url

File: Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ProfessionalController.java (hunk 1 above)
Comment (logikcode) [reply Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ProfessionalController.java:57]: Done

**Review by Jayhmz — COMMENTED** [review]
just fix your merge conflict and you are good to o

**Pull request closed without merging**